docs/custom-metric-filter.md

```diff
@@ -0,0 +1,166 @@
+# ACM Custom Metric Filtering
+
+This guide explains how to adjust the custom metrics that Config Sync
+exports to Prometheus, Cloud Monitoring (formerly known as Stackdriver), and
+Monarch (Google's internal metrics aggregator).
+
+## Background
+
+When Config Sync is installed on a GKE cluster where a
+[default service account](http://cloud/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform)
+is available, or on a GKE cluster that has
+[Workload Identity](http://cloud/kubernetes-engine/docs/how-to/workload-identity)
+enabled and
+[IAM is correctly setup](https://cloud.google.com/anthos-config-management/docs/how-to/monitoring-config-sync#custom-monitoring),
+Config Sync will try to apply the Open Telemetry Collector ConfigMap that contains
+[googlecloud exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter)
+to write metrics into Cloud Monitoring. The default configuration contains a
+bare minimum list of metrics, but some customers might want to configure
+different backends or fine tune the metric list for better monitoring solutions.
+
+The Otel Collector deployment in Config Sync accepts three types of ConigMap
+when configuring pipelines:
+
+1. `otel-collector`
+
+   This is the default configuration that Config Sync deploys in a non-GKE environment, which only exports the metrics through Prometheus exporter
+2. `otel-collector-googlecloud` (or `otel-collector-stackdriver` before Config Sync v1.12.0)
+
+   Config Sync automatically deploys this configuration when `default` credential is available, usually on a GKE cluster. 
```

Comment:
"default" is an impl detail in the gcloud sdk. 
Let's just say "Workload Identity is configured" instead, so it's clearer for users. Yes, you can inject them manually, but that's effectively what Fleet Workload Identity does. So we can just call it Workload Identity.

Reply:
Done